plugins/woocommerce/src/Admin/PluginsHelper.php

```diff
@@ -285,7 +319,7 @@ public static function schedule_install_plugins( $plugins ) {
 		}
 
 		$job_id = uniqid();
-		WC()->queue()->schedule_single( time() + 5, 'woocommerce_plugins_install_callback', array( $plugins, $job_id ) );
+		WC()->queue()->schedule_single( time() + 5, 'woocommerce_plugins_install_callback', array( $plugins ) );
```

Comment:
Good catch! I guess this wasn't used before?

Reply:
Yup! $job_id was never used in the callback function.